include/toplevel.mk

```
@@ -101,12 +101,16 @@ prepare-tmpinfo: FORCE
 
 ifneq ($(DISTRO_PKG_CONFIG),)
 scripts/config/mconf: export PATH:=$(dir $(DISTRO_PKG_CONFIG)):$(PATH)
+scripts/config/nconf: export PATH:=$(dir $(DISTRO_PKG_CONFIG)):$(PATH)
 endif
 scripts/config/mconf:
 	@$(_SINGLE)$(SUBMAKE) -s -C scripts/config all CC="$(HOSTCC_WRAPPER)"
 
 $(eval $(call rdep,scripts/config,scripts/config/mconf))
 
+scripts/config/nconf:
+	@$(_SINGLE)$(SUBMAKE) -s -C scripts/config all CC="$(HOSTCC_WRAPPER)"
```

```suggestion
	@$(_SINGLE)$(SUBMAKE) -s -C scripts/config nconf CC="$(HOSTCC_WRAPPER)"
```
Don't add `nconf` to `all`, it should be its own target.  See comment below.

---

Aaaa, yes, this is i've missed out!